content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/detect-log4shell-demo/index.md

```diff
@@ -86,45 +85,53 @@ As you did with the IAM group, copy the `NAME` value of the cluster you want to
 export CLUSTER=$COPIED_NAME
```

Comment:
We don't use this variable that I can see.

Reply:
Great catch, that's a holdover from the initial version